api/envoy/extensions/filters/http/ext_authz/v3/ext_authz.proto

```diff
@@ -272,8 +274,6 @@ message AuthorizationRequest {
   //   <envoy_v3_api_field_extensions.filters.http.ext_authz.v3.ExtAuthz.with_request_body>`
   //   setting) hence the value of its ``Content-Length`` reflects the size of its payload size.
   //
-  //   This field has been deprecated in favor of :ref:`allowed_headers
-  //   <envoy_v3_api_field_extensions.filters.http.ext_authz.v3.ExtAuthz.allowed_headers>`.
```

Comment:
apologies i didnt realize that there was a message - i was following up a slack question - not sure if its better where it was originally (we defo need a better solution for documenting deprecation)